ddtrace/_trace/utils_botocore/span_pointers/dynamodb.py

```diff
@@ -63,20 +112,27 @@ def _extract_span_pointers_for_dynamodb_response(
             dynamodb_primary_key_names_for_tables, request_parameters
         )
 
-    if operation_name in ("UpdateItem", "DeleteItem"):
+    elif operation_name in ("UpdateItem", "DeleteItem"):
```

Comment:
Just switching to `elif`'s for readability yeah?

Reply:
yup